test/debugger/testParams.json

```diff
@@ -0,0 +1,9 @@
+{ "DebuggerTestParameters": [
```

Comment:
Usually what we do is have a `setupXTest` file and a `X-test.js` file, where the `setup` file contains everything you've put into your `.test` file, and returns a `runTest` function that is parameterized by filename + args, then the `.test` file contains a list of `it(<name>, () => runTest(<args>)` calls.

See https://github.com/facebook/prepack/blob/master/test/react/setupReactTests.js, https://github.com/facebook/prepack/blob/master/test/react/ReactDOM-test.js